src/manifold_dual_contouring.rs

```diff
@@ -396,10 +397,12 @@ fn subsample_octtree<S: RealField + Float + From<f32>>(base: &[Vertex<S>]) -> Ve
     result
 }
 
+#[cfg(not(target_arch="wasm32"))]
```

Comment:
Instead of removing the full Timer struct (and all print statements below), how about modifying Timer for wasm32 in a way that makes it always return std::time::Duration::zero()? (And adding a comment about that to the Timer code)...

Reply:
Sure thing, sounds like a good idea!

Reply:
I've replaced `std::time::Instant` with `instant::Instant` which is a portable layer that emulates `std::time::Instant`'s functionality IFF:
1) the `stdweb` or `wasm-bindgen` feature is enabled, **and**
2) you are targetting wasm32

This appears the be the recommended approach based on my reading of the topic.

If you don't approve of this solution, let me know and I'll go with your suggestion of returning `std::time::Duration::zero()`.

Reply:
Sorry for the late reply.
instant::Instant seems like a cool solution!
As for par_iter() vs. iter() - ideally I'd like to see a transparent wrapper, similar to instant::Instant...one that maps .par_iter() to plain .iter() for WASM. Maybe add a TODO in that regard?